blocks/commerce-cart/commerce-cart.js

```diff
@@ -94,6 +101,73 @@ export default async function decorate(block) {
 
   toggleEmptyCart(isEmptyCart);
 
+  // Handle Edit Button Click
+  async function handleEditButtonClick(cartItem) {
+    try {
+      // Create mini PDP content
+      const miniPDPContent = await createMiniPDP(
+        cartItem,
+        (_updateData) => {
+          // Show success message when mini-PDP updates item
+          const productName = cartItem.name
+            || cartItem.product?.name
+            || placeholders?.Global?.CartUpdatedProductName;
+          const message = placeholders?.Global?.CartUpdatedProductMessage?.replace(
+            '{product}',
+            productName,
+          );
+
+          UI.render(InLineAlert, {
```

Comment:
To avoid potential side effects on the virtual DOM, it might be better to remove the render entirely instead of clearing the inner HTML of `$notification`.

You can use the returned render:

```js
const inlineAlert = await UI.render(InLineAlert, {...});
```

and remove:

```js
inlineAlert.remove();
```

Reply:
It's updated, thanks!